templates/ios/Example/PROJECT.xcodeproj/project.pbxproj

```diff
@@ -208,8 +208,8 @@
 			isa = PBXProject;
 			attributes = {
 				CLASSPREFIX = CPD;
-				LastUpgradeCheck = 0510;
-				ORGANIZATIONNAME = "PROJECT_OWNER";
+				LastUpgradeCheck = 0720;
+				ORGANIZATIONNAME = PROJECT_OWNER;
```

Comment:
Thank you, merged

On Tue, Dec 22, 2015 at 9:26 AM, Orta <notifications@github.com> wrote:

> In templates/ios/Example/PROJECT.xcodeproj/project.pbxproj
> <https://github.com/CocoaPods/pod-template/pull/143#discussion_r48256866>:
>
> > @@ -208,8 +208,8 @@
> >  			isa = PBXProject;
> >  			attributes = {
> >  				CLASSPREFIX = CPD;
> > -				LastUpgradeCheck = 0510;
> > -				ORGANIZATIONNAME = "PROJECT_OWNER";
> > +				LastUpgradeCheck = 0720;
> > +				ORGANIZATIONNAME = PROJECT_OWNER;
>
> I sent a PR adding these [image: :+1:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CocoaPods/pod-template/pull/143/files#r48256866>.
>